auth/client/src/test/java/org/wildfly/security/auth/client/DefaultSSLContextEmptyPathTest.java

```diff
@@ -22,25 +22,30 @@
 import org.junit.Test;
 
 import javax.net.ssl.SSLContext;
+import javax.validation.constraints.AssertTrue;
+
 import java.security.NoSuchAlgorithmException;
 import java.security.Security;
 
 /**
  * Test that default SSLContext provider will throw an exception when configured file path is empty
  */
 public class DefaultSSLContextEmptyPathTest {
-
-    @Test(expected = IllegalArgumentException.class)
+    
+    @Test
     public void defaultSSLContextNonexistentConfigFileTest() {
         Security.insertProviderAt(new WildFlyElytronClientDefaultSSLContextProvider(""), 1);
         Assert.assertNotNull(Security.getProvider("WildFlyElytronClientDefaultSSLContextProvider"));
         AuthenticationContext authenticationContext = AuthenticationContext.captureCurrent();
+
         authenticationContext.run(() -> {
-            try {
-                SSLContext.getDefault();
-            } catch (NoSuchAlgorithmException e) {
-                Assert.fail("Obtaining default SSL context from provider with invalid path threw incorrect exception");
-            }
+            Assert.assertThrows(IllegalArgumentException.class, () -> {
+                try {
+                    SSLContext.getDefault();
+                } catch (NoSuchAlgorithmException e) {
+                    Assert.fail("Obtaining default SSL context from provider with invalid path threw incorrect exception");
+                } 
```

Comment:
Same here, it has a trailing space after the `}` . You can just remove that space